app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    done_lst_path = command.split()
+    if done_lst_path[1] == done_lst_path[2]:
+        return
+    with (open(done_lst_path[1], "r") as file_in,
+          open(done_lst_path[2], "w") as file_out):
+        read_all = file_in.read()
+        file_out.write(read_all)
```

Comment:
Everything is fine!
But If I were you I would change this code to file_out.write(file_in.read())
And check the checklist, there was said: "Make sure you check that the cp input command is correct."